urls.py

```diff
@@ -90,7 +90,7 @@
     url(r'^remote_link/', include('corehq.apps.linked_domain.urls')),
     url(r'^translations/', include('corehq.apps.translations.urls')),
     url(r'^submit_feedback/$', submit_feedback, name='submit_feedback'),
-    url(r'^widget/', include('corehq.apps.widget.urls')),
+    url(r'^integration/', include('corehq.apps.integration.urls')),
```

Comment:
The integration urls are already included in `domain.urls`, which is correct for your new code since `DialerSettingsView` is domain-specific (I didn't notice on first review). I think you can just remove this and everything will continue to work, just with a different URL.

https://github.com/dimagi/commcare-hq/blob/47a7dd6948f49b0dd9e7a5aeca1b795b348cef48/corehq/apps/domain/urls.py#L182